spec/factories/data_imports.rb

```diff
@@ -3,5 +3,10 @@
     description { "Here is a description" }
     user
     file { Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", "files", "occupation-standards-template.xlsx"), "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet") }
+
+    factory :data_import_for_hybrid do
+      description { "Here is a description for this Hybrid w/ Max + Min Hours" }
+      file { Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", "files", "comp-occupation-standards-template.xlsx"), "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet") }
```

Comment:
Should we change the title of this file away from "comp", as it sounds like it is a competency-based template instead of a hybrid example?

Reply:
@littleforest I took the name from Notion, but not sure if those are correct

![image](https://user-images.githubusercontent.com/1335038/219095796-b15a7034-5649-4034-85e6-b89a21e49b65.png)

Reply:
Ah okay! We can always rename later if there is confusion.